src/DelphiRest.inc

```diff
@@ -133,6 +133,46 @@
   {$DEFINE USE_NS}
 {$ENDIF VER290}
 
+{$IFDEF VER300}  // RAD Studio 10 Seattle
+  {$DEFINE DELPHI_7_UP}
+  {$DEFINE DELPHI_2007_UP}
+  {$DEFINE DELPHI_2009_UP}
+  {$DEFINE DELPHI_2010_UP}
+  {$DEFINE DELPHI_XE_UP}
+  {$DEFINE DELPHI_XE2_UP}
+  {$DEFINE DELPHI_XE3_UP}
+  {$DEFINE DELPHI_XE4_UP}
+  {$DEFINE DELPHI_XE5_UP}
+  {$DEFINE DELPHI_XE6_UP}
+  {$DEFINE DELPHI_XE7_UP}
+  {$DEFINE DELPHI_XE8_UP}
+  {$DEFINE USE_NS}
+  {$DEFINE DELPHI_10SEATTLE}
+  {$DEFINE DELPHI_10SEATTLE_UP}
+{$ENDIF}
+
+
+{ Provide compatibility for the next version }
+{$IFDEF CONDITIONALEXPRESSIONS} // Delphi 6+
+  {$IF CompilerVersion >= 30.0} // Delphi 10 Seattle
+    {$DEFINE DELPHI_7_UP}
+    {$DEFINE DELPHI_2007_UP}
+    {$DEFINE DELPHI_2009_UP}
+    {$DEFINE DELPHI_2010_UP}
+    {$DEFINE DELPHI_XE_UP}
+    {$DEFINE DELPHI_XE2_UP}
+    {$DEFINE DELPHI_XE3_UP}
+    {$DEFINE DELPHI_XE4_UP}
+    {$DEFINE DELPHI_XE5_UP}
+    {$DEFINE DELPHI_XE6_UP}
+    {$DEFINE DELPHI_XE7_UP}
+    {$DEFINE DELPHI_XE8_UP}
+    {$DEFINE USE_NS}
+    {$DEFINE DELPHI_10SEATTLE_UP}
+  {$IFEND}
+{$ENDIF}
+
```

Comment:
The second block is for any version after Delphi 10 Seattle... when the next version is released, the code will still compile and work without changes.  Otherwise, you have to remember to alter this include file before someone can use the next Delphi version.   (Which wasn't done for Delphi 10 Seattle.)